src/css/App.less

```diff
@@ -22,13 +22,13 @@
 
 // Override for Design System.
 // Fix for spacing in mega menu headings.
-.o-mega-menu_group-heading,
-.o-mega-menu_content-2-list__featured ul {
+.o-mega-menu__group-heading,
+.o-mega-menu__content-2-list--featured ul {
```

Comment:
Unrelated to this PR, but we shouldn't be adjusting the mega-menu from within CCDB. This should all be removed and moved to https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/ccdb-search/css/main.less

Reply:
Can we just delete then?

Reply:
Yeah maybe so. This and below.

Reply:
All mega menu references have been removed